configs/example/apu_se.py

```diff
@@ -380,6 +380,28 @@
     help="Gfx version for gpuNote: gfx902 is not fully supported by ROCm",
 )
 
+parser.add_argument(
+    "--tcp-rp",
+    type=str,
+    default="TreePLRURP",
+    help="cache replacement policy" "policy for tcp",
```

Comment:
I am very late to the party, but this is how the PR should have been implemented to avoid hardcoding:

Add this line to each of these new arguments:
        choices=ObjectList.rp_list.get_names(),